### PR TITLE
Fix line ending inconsistencies for Zap Scan HTML Report markdown

### PR DESCRIPTION
**Pull request by jtwillis92**
### Summary of Changes
Merging this should hopefully fix the constant re-appearance of `docs/Technical-Documentation/Zap-Scan-HTML-Report.md` in git diffs. It appears that this file was committed with Windows line endings:
```
❯ file docs/Technical-Documentation/Zap-Scan-HTML-Report.md                                                                                                                                                     ─╯
docs/Technical-Documentation/Zap-Scan-HTML-Report.md: ASCII text, with CRLF line terminators
```
Most of us use Mac or Linux, and I assume have set `git config --global core.autocrlf input` - however, in our .gitattributes we had that value set to `auto` which overrides whatever the user setting is. [ref](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings) This setting is the Windows friendly default, and lets git handle line endings how it sees fit - but in our case this conflicts with other declarations in .gitattributes that explicitly set files to use `LF` line endings. If we instead use `input` it will be converted to `LF` only on commit and then on checkout the line endings will be processed using the `core.eol` setting which defaults to `native` and will be handled as needed per OS. [Additional ref](https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/#:~:text=your%20working%20directory.-,core.,file%20to%20the%20object%20database.)

To convert the file, I ran the following locally and then committed the change:
```
❯ dos2unix docs/Technical-Documentation/Zap-Scan-HTML-Report.md                                                                                                                                                 ─╯
dos2unix: converting file docs/Technical-Documentation/Zap-Scan-HTML-Report.md to Unix format...
```

### How to Test
* Check out this branch on your local machine
* Confirm that `docs/Technical-Documentation/Zap-Scan-HTML-Report.md` does not show as a changed file and has the correct line endings:
```
❯ file docs/Technical-Documentation/Zap-Scan-HTML-Report.md                                                                                                                                                     ─╯
docs/Technical-Documentation/Zap-Scan-HTML-Report.md: ASCII text
```

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [x] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [x] Check against the criteria:
  - [x] Zap scan doc does not show as a changed file due to line ending differences on checkout

As facilitator/product manager, @kniz-raft will decide if ACs are met from Raft's perspective.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

N/A repo config change

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

N/A repo config change

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*

N/A repo config change

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful build with a single command*

N/A repo config change

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

N/A repo config change

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

N/A repo config change

### CODE REVIEW

**Review by jorgegonzalez — APPROVED** [review]
LGTM 👍

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Set default behaviour, in case users don't have core.autocrlf set.
-* text=auto
+* text=input
```
Comment (jtwillis92) [inline .gitattributes:2]: We want to always convert to LF on commit to Git but leave it up to eol settings declared below and the native system to handle checkout. This should prevent this issue from coming up again in the future, as we won't be able to have files committed with CRLF endings as seen in the other file change in this PR.

File: docs/Technical-Documentation/Zap-Scan-HTML-Report.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-![image](images/circleci.png)
-
-3. Click the `Artifacts` tab at the top of the screen once the process has completed.
-
-![image](images/artifacts.png)
-
-4. Click the `owasp_report.html` link to view the report.
-
-![image](images/report.png)
+# OWASP Zap Scan HTML Report
```
Comment (jtwillis92) [inline docs/Technical-Documentation/Zap-Scan-HTML-Report.md:1]: Only change in this file is converting line endings from CRLF to LF.

**Comment by ADPennington** [comment]
nice find @jtwillis92! test results from windows below. 👍🏾 not surprisingly, it took me longer to switch to this branch given the zap md issue than to test this 😄 

```
apennington@HHSL9XYTF13 MINGW64 ~/OneDrive - HHS Office of the Secretary/Desktop/TDP-testing/Raft_TANF-app/TANF-app (raft-tdp-main)
$ git checkout fix-line-endings-zap-doc
Switched to a new branch 'fix-line-endings-zap-doc'
Branch 'fix-line-endings-zap-doc' set up to track remote branch 'fix-line-endings-zap-doc' from 'origin'.

apennington@HHSL9XYTF13 MINGW64 ~/OneDrive - HHS Office of the Secretary/Desktop/TDP-testing/Raft_TANF-app/TANF-app (fix-line-endings-zap-doc)
$ git status
On branch fix-line-endings-zap-doc
Your branch is up to date with 'origin/fix-line-endings-zap-doc'.

nothing to commit, working tree clean

apennington@HHSL9XYTF13 MINGW64 ~/OneDrive - HHS Office of the Secretary/Desktop/TDP-testing/Raft_TANF-app/TANF-app (fix-line-endings-zap-doc)
$ file docs/Technical-Documentation/Zap-Scan-HTML-Report.md
docs/Technical-Documentation/Zap-Scan-HTML-Report.md: ASCII text

apennington@HHSL9XYTF13 MINGW64 ~/OneDrive - HHS Office of the Secretary/Desktop/TDP-testing/Raft_TANF-app/TANF-app (fix-line-endings-zap-doc)
$

```

**Review by abottoms-coder — APPROVED** [review]

**Review by ADPennington — APPROVED** [review]
thanks @jtwillis92 👍🏾 @abottoms-coder this is ready to merge. 

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [x] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [x] Check against the criteria:
  + [x] Zap scan doc does not show as a changed file due to line ending differences on checkout

As Product Owner, @lfrohlich will decide if ACs are met.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

_n/a: repo config changes only_

### Deliverable 3: Properly Styled Code

_n/a: repo config changes only_

### Deliverable 4: Accessible

_n/a: repo config changes only_

### Deliverable 5: Deployed

_n/a: repo config changes only_

### Deliverable 6: Documented

_n/a: repo config changes only_

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

_no new security issues detected_

**Pull request merged**